apps/extension/src/test/init.ts

```diff
@@ -0,0 +1,76 @@
+import { KVStore } from "@anoma/storage";
+import { Chain } from "@anoma/types";
```

Comment:
I would remove this import if it's not going to be used in this file, same for `KeyStore` below